app/bundles/LeadBundle/Entity/PointsChangeLog.php

```diff
@@ -87,10 +87,8 @@ public static function loadMetadata(ORM\ClassMetadata $metadata): void
 
     /**
      * Get id.
-     *
-     * @return int
      */
-    public function getId()
+    public function getId(): int
```

Comment:
In case of these IDs, I'm not 100% certain this is the right thing to do. As the IDs are BIGINT in MySql which can have greater value than what PHP can fit in the `int` type. So we shouldn't retype it to int, actually.

If we set the return type to `int` then we make it official and hard to change later as it will be another BC break. How would you recommend to proceed?